logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -69,6 +79,33 @@ def get_trace_id_from_flask():
     return trace_id
 
 
+def get_trace_id_from_webapp2():
+    """Get trace_id from webapp2 request headers.
+
+    :rtype: str
+    :return: Trace_id in HTTP request headers.
```

Comment:
OK, I was just copying from what the other functions in this file did.  I'll fix them all.  (Similarly for trace_id below.)